sdk/attestation/azure-security-attestation/README.md

```diff
@@ -193,52 +191,51 @@ If the attestation service instance is running in Isolated mode, the set_policy
 Under the covers, the SetPolicy APIs create a [JSON Web Token][json_web_token] based on the policy document and signing information which is sent to the attestation service.
 
 ```python
-    policy_set_response = attest_client.set_policy(AttestationType.SGX_ENCLAVE,
-        attestation_policy,
-        signing_key=AttestationSigningKey(key, signing_certificate))
-    policy_get_response = attest_client.get_policy(AttestationType.SGX_ENCLAVE)
-    assert policy_get_response.value == attestation_policy
+policy_set_response = attest_client.set_policy(AttestationType.SGX_ENCLAVE,
+    attestation_policy,
+    signing_key=key,
+    signing_certificate=signing_certificate)
+new_policy, _ = attest_client.get_policy(AttestationType.SGX_ENCLAVE)
+assert new_policy == attestation_policy
```

Comment:
I would remove the assert from the sample, normally assert statements are reserved for test code